src/main/events/catalogue/get-game-shop-details.ts

```diff
@@ -4,7 +4,29 @@ import { getSteamAppDetails } from "@main/services";
 import type { ShopDetails, GameShop, SteamAppDetails } from "@types";
 
 import { registerEvent } from "../register-event";
-import { searchRepacks } from "../helpers/search-games";
+
+const getLocalizedSteamAppDetails = (
+  objectID: string,
+  language: string
+): Promise<ShopDetails | null> => {
+  const englishAppDetails = getSteamAppDetails(objectID, "english");
+
+  if (language === "english") return englishAppDetails;
+
+  return Promise.all([
+    englishAppDetails,
+    getSteamAppDetails(objectID, language),
+  ]).then(([appDetails, localizedAppDetails]) => {
+    if (appDetails && localizedAppDetails) {
+      return {
+        ...localizedAppDetails,
+        name: appDetails.name,
```

Comment:
return the getSteamAppDetails(objectID, "english") when language is english and get the game from steamGameRepository because we already have the game name in english and the objectId of steam game, that will saved us to fetch two time with the only purpose to get the name in english, something we already have in DB

![image](https://github.com/hydralauncher/hydra/assets/167036558/9894cf30-c31a-4664-8ed2-9e23fdc66ace)

Reply:
Nice suggestion. Will change.